README.md

```diff
@@ -1,22 +1,48 @@
 2023/03/01
 Hsiang-He Lee (lee1061@llnl.gov)
+2023/05/24 updated
 
-E3SM chemistry diagnostic package version1.0 (ChemDyg v1.0) is designed for E3SM chemistry development. There are total 11 types of plot and 3 types of table for model-to-model and model-to-observation comparison.
+E3SM chemistry diagnostic package version 0.1.0 (ChemDyg v0.1.0) is designed for E3SM chemistry development. There are total 11 types of plot and 3 types of table for model-to-model and model-to-observation comparison.
 
 Index page and example figures:
 https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/ac.lee1061/20220914.PAN.MZThet.v2.LR.bi-grid.amip.chemUCI_Linozv3/e3sm_chem_diags/plots/
 
 ChemDyg is executed by zppy, which is a post-processing toolchain for E3SM written in Python (https://github.com/E3SM-Project/zppy).
 
-*How to run ChemDyg via zppy script on chrysalis?
-1. Setup environment 
->source /gpfs/fs1/soft/chrysalis/manual/anaconda3/2020.11/etc/profile.d/conda.sh
->conda activate zppy_dev_chem
+# How to run ChemDyg via zppy script on chrysalis?
 
-2. Download zppy and ChemDyg on your working folder 
+*Clone repository
 
-3. Modify .cfg run script and then
->zppy -c ChemDyg_example_script.cfg
+```git clone git@github.com:E3SM-Project/zppy.git```
+
+```cd zppy```
+
+*Checkout branch
+```git checkout golaz/plugins```
+
+*Create environment
+
+```module load anaconda3/2020.11```
+
+```conda env create --name zppy_dev_chem -f conda/dev.yml```
+
+*Activate
+
+```source /gpfs/fs1/soft/chrysalis/manual/anaconda3/2020.11/etc/profile.d/conda.sh```
+
+```conda activate zppy_dev_chem```
+
+*Install chemdyg package
+
+```install -y -c e3sm/label/chemdyg_dev chemdyg```
```

Comment:
This was a temporary way to install that I don't think we want to publish for users.

Reply:
````suggestion
```
conda install -y chemdyg
```
````